.agents/skills/bifrost-setup/SKILL.md

```
@@ -0,0 +1,124 @@
+## Login
+
+```bash
+bifrost login --url {url}
+```
+
+This opens a browser for authentication and saves credentials to `~/.bifrost/credentials.json`.
+
+## MCP Configuration
+
+Check existing configuration:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)